git-guides/install-git.md

```diff
@@ -0,0 +1,42 @@
+# Install Git
+
+How to install Git on any OS
+
+Git can be installed on the most common operating systems like Windows, Mac, and Linux. In fact, Git comes by default on most Mac and Linux machines!
+
+<!--- Can we use javascript to automatically detect OS and create quick access buttons for “download on Mac”, “Install on Mac” etc.
+- Nav list for quick navigation to the following sections.-->
+
+## Checking for Git
+
+To see if you already have Git installed, open up your terminal application.
+
+- If you're on a Mac, look for an appliction called "Terminal".
+- If you're on a Windows machine, look for an application called "Git Bash". 
+
+Once you've opened your terminal application, type `git version`. The output will either tell you which version of Git is installed, or it will alert you that `git` is an unknown command. If it's an unknown command, read further and find out how to install Git.
+
+## GitHub Desktop
+
+Regardless of if you have Git installed or not, GitHub Desktop offers a simple collaboration tool for Git. Installing GitHub Desktop also installs Git, if you don't already have it. You can [learn more here](https://desktop.github.com/).
+
+## Install Git on Windows
+
+[`git-scm`](https://git-scm.com/download/win) is the most recommended resource for downloading Git for Windows. The advantage of downloading Git from `git-scm` is that your download will include the recommended terminal, `Git Bash`. If you aren't sure which options to select during the install, we recommend keeping with the defaults.
+
+## Install Git on Mac
+
+Most versions of MacOS will already have `git` installed, and you can activate it through the terminal with `git version`. However, if you don't have Git installed for whatever reason, you can [download it here](https://git-scm.com/download/mac).
+
+## Install Git on Ubuntu
+
+- Short list of the most popular methods for installing Git on Ubuntu.
+- Can link out to more comprehensive resources for installation instructions.
+
+## Install Git on Linux
+
+Fun fact: Git was originally developed to version the Linux operating system! So, it only makes sense that it is easy to configure to run on Linux. You can download the proper Git versions and read more about how to install on specific Linux systems, like installing Git on Ubuntu, [in git-scm's documentation](https://git-scm.com/download/linux).
```

Comment:
@brianamarie we can go a little more in depth on this page – these sections would probably be a good place to give more actionable steps for installing Git. i.e. If you have checked for Git and it's not installed, here's the easiest way to install it on Mac (with a couple steps) and potentially other leading alternatives (homebrew, or reiterating the CTA for GitHub Desktop)

Reply:
@DylanAStark Absolutely - but I worry here about duplicating the exact documentation from the other sites we recommend, like git-scm. Is that level of verbatim duplication what we're going for here?

Reply:
@brianamarie I think it's definitely okay for them to be similar, especially because they're serving the same purpose 👍 though our goal is come across not as "documentation" but as a user-friendly resource to quickly get started, so too exhaustively covering methods is probably unhelpful as well – looking for a happy medium